frontend/src/components/App/TopBar.tsx

```diff
@@ -87,6 +88,34 @@ export default function TopBar({}: TopBarProps) {
   const history = useHistory();
   const { appBarActions, appBarActionsProcessors } = useAppBarActionsProcessed();
 
+  // Fetch current user info from backend
+  const [me, setMe] = React.useState<{ username?: string; email?: string } | null>(null);
+  React.useEffect(() => {
```

Comment:
Naming the function used inside the useEffect would make it a bit more readable.